docs/quickstart.md

```diff
@@ -46,10 +47,7 @@ If you don't like `npm` or have trouble installing the tool, you can manually cr
   <head>
     <meta name="viewport" content="width=device-width,initial-scale=1" />
     <meta charset="UTF-8" />
-    <link
-      rel="stylesheet"
-      href="//cdn.jsdelivr.net/npm/docsify@4/themes/vue.css"
-    />
+    <link rel="stylesheet" href="//cdn.jsdelivr.net/npm/docsify@5/themes/vue.min.css" />
```

Comment:
Everything looks OK to me. But until we release v5, these links are broken.

Reply:
Yes, but I'm not sure why that matters. The docs in the `develop` branch already contain new features and changes that will only be available when v5 is released.

- These changes will not be visible on [docsify.js.org](https://docsify.js.org) until `develop` is merged into `main`.
- These `@5/dist` URLs are not broken during local development or Vercel previews because CDN URLs are [rewritten](https://github.com/docsifyjs/docsify/blob/1ceb03b013e1454c0fe3a76971c49bdde00944e6/middleware.js#L8-L30) to `/dist` in those environments. For example, notice how this PR contains `@5/dist` CDN URLs in [`/docs/index.html`](https://github.com/docsifyjs/docsify/blob/v5-build-overhaul/docs/index.html) but the Vercel [preview](https://github.com/docsifyjs/docsify/blob/v5-build-overhaul/docs/index.html) works as expected.